docs/terms/expectation.md

```diff
@@ -3,10 +3,134 @@ title: Expectation
 id: expectation
 hoverText: A verifiable assertion about data.
 ---
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '/docs/images/universal_map/_um_setup_header.mdx'
+import ConnectHeader from '/docs/images/universal_map/_um_connect_header.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
 
 An Expectation is a verifiable assertion about data. 
 
+### Features and promises
+
+Great Expectations is a framework for defining Expectations and running them against your data. Like assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behavior. Unlike traditional unit tests, Great Expectations applies Expectations to data instead of code. For example, you could define an Expectation that a column contain no null values, and Great Expectations would run that Expectation against your data, and report if a null value was found.
+
+Expectations *enhance communication* about your data and *improve quality* for data applications. Using Expectations helps reduce trips to domain experts and avoids leaving insights about data on the "cutting room floor."
+
+### Relationship to other objects
+
+Expectations are grouped into <TechnicalTag relative="../" tag="expectation_suite" text="Expectation Suites" />, which are in turn stored in an <TechnicalTag relative="../" tag="expectation_store" text="Expectation Store" />.  <TechnicalTag relative="../" tag="profiler" text="Profilers" /> will analyze data in order to generate Expectations, and <TechnicalTag relative="../" tag="checkpoint" text="Checkpoints" /> rely on Expectation Suites (and the Expectations contained therein) to <TechnicalTag relative="../" tag="validation" text="Validate" /> data.
+
+## Use cases
+
+<CreateHeader/>
+
+Expectations are obviously a fundamental component of the Create Expectations step in working with Great Expectations.  It is when you are creating Expectations that you will have the most direct interaction with them.  For further information on this process, please see [our overview on the Create Expectations process](../guides/expectations/create_expectations_overview.md), and [our related how-to guides](../guides/expectations/index.md).
```

Comment:
You may want a note in here about editing Expectations. Expectations are not meant to be static - they will be iterated on as data and the understanding of those change, and so they may need to be interacted with after Creation.

Reply:
I've added this mention.  It will show up in the next commit.